### PR TITLE
Cumulus 1636 cma fix

### PR DESCRIPTION
**Pull request by Jkovarik**
Addresses [CUMULUS-1636](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1636)

## Changes

Update lambda configurations to take advantage of price/performance at higher memory levels for lambdas not using external resources.  See ticket for more details. 

This PR will include updating to CMA 1.2.1 upon approval of that pr: 

https://github.com/nasa/cumulus-message-adapter/pull/72

### CODE REVIEW

File: example/cumulus-tf/python-reference-task.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ resource "aws_lambda_function" "python_reference_task" {
   role             = module.cumulus.lambda_processing_role_arn
   runtime          = "python3.7"
   timeout          = 300
-  memory_size      = 256
+  memory_size      = 1536
```
Comment (laurenfrederick) [inline example/cumulus-tf/python-reference-task.tf:9]: Why is one 1536 and one 1556?

Comment (Jkovarik) [reply example/cumulus-tf/python-reference-task.tf:9]: 1556 was a typo (1024*1.5 == 1536).

**Comment by Jkovarik** [comment]
Updated to reference a build from the just-deployed CMA v1.2.1, will merge once tests  pass.

**Comment by Jkovarik** [comment]
Closing as the current CMA will not work with Java lambdas due to a packaging annoyance.

**Pull request closed without merging**